tfhe/src/core_crypto/mod.rs

```diff
@@ -17,3 +17,6 @@ pub mod prelude;
 pub mod seeders;
 
 pub mod fft_impl;
+
+#[cfg(any(test, doctest, feature = "internal-keycache"))]
+pub mod keycache;
```

Comment:
no, only for internal-keycache enabled, otherwise we have the keycache on for all tests

Reply:
I need to mention at least `test`otherwise `pcc` command fails.

Reply:
still not good enough, there must be some missing internal-keycache stuff then in the tests, we must be able to run without keycache

Reply:
Ok I'll look at it.